x-pack/plugins/aiops/public/embeddable/change_point_chart/initialize_chane_point_controls.ts

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { StateComparators } from '@kbn/presentation-publishing';
+import fastIsEqual from 'fast-deep-equal';
+import { BehaviorSubject } from 'rxjs';
+import type { ChangePointDetectionViewType } from '@kbn/aiops-change-point-detection/constants';
+import type { ChangePointComponentApi, ChangePointEmbeddableState } from './types';
+
+type ChangePointEmbeddableCustomState = Omit<ChangePointEmbeddableState, 'timeRange'>;
+
+export const initializeChangePointControls = (rawState: ChangePointEmbeddableState) => {
```

Comment:
Typo in the filename here `initialize_chane_point_controls.ts`

Reply:
fixed in https://github.com/elastic/kibana/pull/182246/commits/bcef461d85cd183ca3576fea2173aaea67c8878d